src/components/userPreferencesModal/HotKeysPreferences.styl

```diff
@@ -0,0 +1,3 @@
+.col-md-6
```

Comment:
You may want to rip this out and update the preferences modal to use flexbox

Reply:
Ok